arduino-ide-extension/package.json

```diff
@@ -170,17 +171,17 @@
     }
   ],
   "arduino": {
-    "cli": {
+    "arduino-cli": {
```

Comment:
This line must also be updated to reflect the key name change:

https://github.com/arduino/arduino-ide/blob/28bc470f6b13b87f1c60606c06b7d65a3c82bd5f/arduino-ide-extension/src/node/config-service-impl.ts#L135

Otherwise the "CLI Version" field in the "About" dialog is empty:

![image](https://github.com/arduino/arduino-ide/assets/8572152/022f6f5a-081b-4d51-b6d4-d11069d0ef48)

Reply:
Fixed by https://github.com/arduino/arduino-ide/commit/75480bd53f247fb214075339a6a55cf757adaab5

Thanks!